src/libConsensus/ConsensusLeader.cpp

```diff
@@ -177,20 +220,29 @@ void ConsensusLeader::StartConsensusSubsets() {
   }
 
   m_numSubsetsRunning = m_consensusSubsets.size();
-  for (unsigned int index = 0; index < m_consensusSubsets.size(); index++) {
+  // subset 0 last to be started. giving community nodes the advantage
+  for (unsigned index = m_consensusSubsets.size(); index > 0; index--) {
     // If overall state has somehow transitioned from CHALLENGE_DONE or
     // FINALCHALLENGE_DONE then it means consensus has ended and there's no
     // point in starting another subset
     if (m_state != CHALLENGE_DONE && m_state != FINALCHALLENGE_DONE) {
       break;
     }
-    ConsensusSubset& subset = m_consensusSubsets.at(index);
+
+    // delay starting every subset to avoid network congestion
+    if (index < m_consensusSubsets.size()) {
+      LOG_GENERAL(INFO, "Waiting for "
```

Comment:
Can make it shorter, like "[SubsetID: X] Waiting X seconds"?  Recently I am a fan of shorter messages.

Reply:
Sure :)